forgae-shape-vue-command.md

```diff
@@ -0,0 +1,192 @@
+
+Let's get the deployment details with the following command:
+```
+forgae history
+```
+As a result, we can see a table with the most important details. For example:
+
+```
+┌───────────────┬──────────────────────────────────────────────────────┐
+│ Event Time    │ 2 May, 15:05:17                                      │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Executor      │ Deployer                                             │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Name or Label │ ToDoManager                                          │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Tx Hash       │ undefined                                            │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Status        │ Success                                              │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Gas Price     │ 1000000000                                           │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Gas Used      │ 4000                                                 │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Result        │ ct_HVb6d4kirgqzY1rShmzRTRwukcsXobjHcpLVD2EggoHmn6wt2 │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Network ID    │ ae_devnet                                            │
+└───────────────┴──────────────────────────────────────────────────────┘
+```
+Copy the address of the contract that is shown as a **Result** in the above report table.
+Assign the deployed contract address to the contractAddress property in ```aepp/src/contractDetails.js``` file.
+
+## Shape project
+
+The shape project is a sample Wallet/Identity Aepp that expects an Aepp to be loaded into an iFrame contained into this base aepp. 
+The most appropriate way to provide identity is the integration between the [base-aepp](https://base.aepps.com/) and your Æpp.
+The alternative identity provider is the custom identity aepp. We will use it just for simplicity and showing the app development process.
+
+## Run the identity/wallet provider aepp
+Let's start the identity/wallet Aepp, which will start on port 8080:
+```
+cd aepp-forgae-shape-vue/identity-provider
+npm run serve
+```
+
+## Run the aepp
+Now we are ready to run the aepp, which will start on port 8081. Open new terminal window and execute the following commands:
+
+```
+cd aepp-forgae-shape-vue/aepp
+npm run serve
+```
+
+## Shape Project appearance
+
+![shape project](https://raw.githubusercontent.com/aeternity/aepp-forgae-shape-vue/master/shape-project.png "Shape project")
+
+
+## Important considerations
+
+
+### Identity provider
+
+#### Get client
+The application flow starts with getting the client. The code placed in the ```created``` lifecycle hook takes care of this.
+
+### ToDo aepp
```

Comment:
I feel a few more details about how this aepp uses the blockchain (in technical terms) would be nice

Reply:
@alekseytsekov will you still work on this?

Reply:
@thepiwo yes